cmake_modules/commonSetup.cmake

```diff
@@ -1,4 +1,4 @@
-################################################################################
+﻿################################################################################
```

Comment:
I think these changes are caused by BOM markers - probably best to avoid them

Reply:
Yes, this change remove the BOM markers.